websites/P/Physics Wallah/presence.ts

```diff
@@ -0,0 +1,147 @@
+const presence = new Presence({
+	clientId: "1134044987277975616",
+});
+
+let mediaTimestamps: [number, number];
+const time = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "phy",
+		},
+		Path = document.location.pathname;
+
+	presenceData.buttons = [
+		{ label: "Open Website", url: document.location.href },
+	];
```

Comment:
We don't allow this. Buttons must not lead to the main page & we prefer them to be specific per page.
```suggestion
```